client/hosting/overview/components/plan-bandwidth.tsx

```diff
@@ -1,10 +1,19 @@
+import { FEATURE_SFTP } from '@automattic/calypso-products';
+import { LoadingPlaceholder } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
 import { convertBytes } from 'calypso/my-sites/backup/backup-contents-page/file-browser/util';
 import { useSiteMetricsQuery } from 'calypso/my-sites/site-monitoring/use-metrics-query';
 import { useSelector } from 'calypso/state';
+import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
+import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
+import siteHasFeature from 'calypso/state/selectors/site-has-feature';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
 import { getSelectedSite } from 'calypso/state/ui/selectors';
 
 interface PlanBandwidthProps {
+	children?: React.ReactNode;
 	siteId: number;
+	className?: string;
 }
```

Comment:
```suggestion
type PlanBandwidthProps = React.PropsWithChildren< {
	siteId: number;
} >;
```
Nit, but I typically advise using `PropsWithChildren` instead of manually typing the `children` prop. Also, it looks like we aren't using the `className` prop, so let's remove that.

Lastly, I'd advise moving this definition closer to where it's used (so just above `PlanBandwidth`)